content/news/027/index.md

```diff
@@ -114,6 +114,23 @@ The code source is freely [available on GitHub](https://github.com/djeedai/libra
 [@djeedai]: https://twitter.com/djeedai
 [Bevy Engine]: https://bevyengine.org/
 
+### [Berry Run] - !!! PLAY IN YOUR BROSWER NOW !!!
```

Comment:
No all-caps, please!

```suggestion
### [Berry Run]
```

I think this kind of info should go in the section itself rather than the title.